nss-day-cohort-47/holiday-road-tundra-travelers
# cc-creatingotherpages

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
There was nothing that this fixed, it only added.

## Type of change

HTML and CSS, mostly gives the potential to add in search functionality later on when connected.

Please delete options that are not relevant.


# Testing Instructions
Look to see if you can see Jake's face next to the title in each page and make sure you can see the search bar in the search page.

there is a new error "Uncaught ReferenceError: closeNav is not defined
    at HTMLAnchorElement.onclick ((index):32)
onclick @ (index):32" being caused. This does not stop the scripts from running, and seems to be incorrect since it is defined.
"A feature not a flaw" - A wise man

# Checklist:
